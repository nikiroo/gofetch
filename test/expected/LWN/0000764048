                 [$] LIFE BEHIND THE TINFOIL CURTAIN               

  [Security] Sep 5, 2018 22:11 UTC (Wed) (jake)

  o Reference: 0000764048
  o News link: https://lwn.net/Articles/764048
  o Source link: 


    [$]   Sorry,   this  article  is  currently  available  to  LWN
    suscribers only [https://lwn.net/subscribe/].



