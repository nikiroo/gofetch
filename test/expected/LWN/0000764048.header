0[$] Life behind the tinfoil curtain	null/LWN/0000764048		70
i  [Security] Sep 5, 2018 22:11 UTC (Wed) (jake)
i
i    Security  and  convenience  rarely go hand-in-hand, but if your
i    job  (or  life) requires extraordinary care against potentially
i    targeted  attacks,  the  security side of that tradeoff may win
i    out.  If  so, running a system like [1]Qubes OS on your desktop
i    or  [2]CopperheadOS  on  your  phone might make sense, which is
i    just  what Konstantin Ryabitsev, Linux Foundation (LF) director
i    of  IT  security,  has done. He reported on the experience in a
i    [3]talk  [YouTube  video]  entitled  "Life  Behind  the Tinfoil
i    Curtain" at the 2018 [4]Linux Security Summit North America .
i    
i    
i    
i    [1] https://www.qubes-os.org/
i    
i    [2] https://copperhead.co/
i    
i    [3] https://www.youtube.com/watch?v=8cU4hQg6GvU&index=6&list=P-
i    LbzoR-pLrL6rOT6m50HdJFYUHyvA9lurI&t=0s
i    
i    [4] https://events.linuxfoundation.org/events/linux-security-s-
i    ummit-north-america-2018/
i
