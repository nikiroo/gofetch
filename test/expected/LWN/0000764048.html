<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>[$] Life behind the tinfoil curtain</h2>
	<div class='details'>([Security] Sep 5, 2018 22:11 UTC (Wed) (jake))</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0000764048</a></li>
        <li>News link: <a href='https://lwn.net/Articles/764048'>https://lwn.net/Articles/764048</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		[$] Sorry, this article is currently available to LWN suscribers only [https://lwn.net/subscribe/].
	</div>
<hr/>
</div>
</body>
